### DISCUSSION TITLE
How to prepare a custom image dataset?

### DISCUSSION THREAD

**Discussion by KirDontsov** [tracel-ai/burn#2325] [Q&A] [answered]
Sorry, but in tutorials and examples I haven't found any information about how to prepare a custom dataset. Only examples how to work with already prepared dataset.
So I'm trying to do it with Python now, but may be there is a better way to do it?

**Comment by nathanielsimard** [comment, marked as answer]
I think you'll find this blog interesting: https://burn.dev/blog/building-blocks-dataset/

**Reply by KirDontsov** [reply]
Thanks a lot!)
